components/x-live-blog-wrapper/readme.md

```diff
@@ -1,11 +1,11 @@
 # x-liveblog-wrapper
 
-This module has these features and scope.
+This module displays a list of live blog posts using `x-live-blog-post` component. Also handles live updates on the live blog.   
```

Comment:
```suggestion
This module displays a list of live blog posts using `x-live-blog-post` component. It also connects to an event stream which provides updates for the list. Based on these update events this component will add, remove and update `x-live-blog-post` components in the list.   
```

This will test if I have been following discussions correctly and understood the component :)